### PR TITLE
Migrate checksum_type configuration for a repo

### PR DESCRIPTION
**Pull request by goosemania**
closes #6813
https://pulp.plan.io/issues/6813

### CODE REVIEW

File: pulp_2to3_migration/app/plugin/rpm/repository.py [hunk 1]
```diff
@@ -55,24 +55,30 @@ def migrate_to_pulp3(cls, pulp2distributor, repo_version):
...
 
         # create distribution
-        pulp2_config = pulp2distributor.pulp2_config
         distribution_data = cls.parse_base_config(pulp2distributor, pulp2_config)
 
         # ensure that the base_path does not end with / in Pulp 3, it's often present in Pulp 2.
         base_path = pulp2_config.get('relative_url', pulp2distributor.pulp2_repo_id)
         distribution_data['base_path'] = base_path.rstrip('/')
         distribution_data['publication'] = publication
+        distribution_data['checksum_type'] = pulp2_config.get('checksum_type', '')
```
Comment (ipanova) [inline pulp_2to3_migration/app/plugin/rpm/repository.py:81]: i don't think this is needed, RpmDistribution does nor have a checsum_type field

File: CHANGES/6813.feature [hunk 2]
```diff
@@ -0,0 +1 @@
+Migrate checksum_type configuration for a publication.
```
Comment (ipanova) [inline CHANGES/6813.feature:1]: s/for a publication/for an Rpm publication

File: pulp_2to3_migration/app/plugin/rpm/repository.py (hunk 1 above)
Comment (goosemania) [reply pulp_2to3_migration/app/plugin/rpm/repository.py:81]: I agree, I believe I removed that. I'll check if it's the recent changes that I pushed :/ thanks

**Comment by ipanova** [comment]
I do not need to re-review

**Pull request merged**